experimental/benchmark/src/main.rs

```diff
@@ -0,0 +1,131 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Trusted Database Benchmark.
+//!
+//! This benchmark measures average gRPC request time to a database that is either:
+//! - Implemented as part of the Trusted Database Oak application (using `oak::OakApplication`)
+//! - Implemented as a `tonic` gRPC server without using Oak (using `vanilla::VanillaApplication`)
```

Comment:
I know I suggested vanilla, but after seeing the reports the code generates, I think `raw` is a better name. Just as a suggestion, up to you :)

Reply:
Done